docs/external_links.rst

```diff
@@ -0,0 +1,15 @@
+Related External Links
+======================
+
+This page lists links to information, tutorial, articles, and posts on Apache Mynewt hosted outside the ASF infrastructure.
+
+.. contents::
+  :local:
+  :depth: 2
+
+Helpful Tutorials and Tools
+---------------------------
+
+- `Sensor app and network tutorial <https://medium.com/@ly.lee/build-your-iot-sensor-network-stm32-blue-pill-nrf24l01-esp8266-apache-mynewt-thethings-io-ca7486523f5d>`__ : A detailed step-by-step guide on how to build your IoT Sensor Network using STM32 Blue Pill, nRF24L01, ESP8266, thethings.io, and Apache Mynewt. 
+
+- `Ubuntu VirtualBox with Source Code and Build Tools <https://github.com/lupyuen/stm32bluepill-mynewt-sensor/releases/tag/v1.0.0>`-- : An Ubuntu VirtualBox with the complete Mynewt 1.6.0 source code + build tools for STM32 Blue Pill to go with the tutorial above. `
```

Comment:
This has to have a `__` (instead of `--`) to correctly.

Reply:
Also since this are external resources, I think it would be better to open them in new tabs/windows instead of moving the user out of the docs site. Restructured text does not have inline syntax for this but it can be done using:

```
- |sensor_app_link| : A detailed step-by-step guide on how to build your IoT Sensor Network using STM32 Blue Pill, nRF24L01, ESP8266, thethings.io, and Apache Mynewt. 

.. |sensor_app_link| raw:: html

   <a href="https://medium.com/@ly.lee/build-your-iot-sensor-network-stm32-blue-pill-nrf24l01-esp8266-apache-mynewt-thethings-io-ca7486523f5d" target="_blank">Sensor app and network tutorial</a>

- |ubuntu_vbox_link| : An Ubuntu VirtualBox with the complete Mynewt 1.6.0 source code + build tools for STM32 Blue Pill to go with the tutorial above. `

.. |ubuntu_vbox_link| raw:: html

   <a href="https://github.com/lupyuen/stm32bluepill-mynewt-sensor/releases/tag/v1.0.0" target="_blank">Ubuntu VirtualBox with Source Code and Build Tools</a>
```

Reply:
Good idea. I will make the changes and modify the PR. thanks!